### PR TITLE
Add excludeGroupInfo function

### PR DESCRIPTION
**Pull request by Powersource**
For https://github.com/ssbc/ssb-tribes2/pull/88

### CODE REVIEW

File: test/tribes.js [hunk 1]
```diff
@@ -338,5 +341,25 @@ test('You can add multiple keys to a group and switch between them for writing',
...
+
+  const groupInfo = await sbot.box2.getGroupInfo(groupId)
+
+  t.deepEquals(
+    groupInfo,
+    { removed: true },
+    'removing group info just leaves removed: true'
+  )
+
+  // TODO: list
```
Comment (Powersource) [inline test/tribes.js:362]: todo

**Comment by Powersource** [comment]
@SocketSecurity ignore

**Comment by Powersource** [comment]
@SocketSecurity ignore-all

File: test/tribes.js [hunk 2]
```diff
@@ -338,5 +341,27 @@ test('You can add multiple keys to a group and switch between them for writing',
...
+
+  t.deepEquals(
+    groupInfo,
+    { excluded: true },
+    'excluding group info just leaves excluded: true'
+  )
+
+  const list = await pull(sbot.box2.listGroupIds(), pull.collectAsPromise())
+
+  t.deepEquals(list, [groupId], 'group is still in list after exclusion')
```
Comment (mixmix) [inline test/tribes.js:364]: same question as other PR - is this here still so you can know you were excluded?

**Review by mixmix — APPROVED** [review]
NICE

ready to go after that other module published

**Review by staltz — APPROVED** [review]
This PR looks simple and easy, we just need to address the same things as said in ssb-keyring:

- `opts` not needed for exclude API
- `list()` shouldn't (?) have excluded group IDs
- Remember to update package.json to use the new version of keyring before merging here

Other than that, no need to ask for another review from me, looks good.

File: test/tribes.js (hunk 2 above)
Comment (Powersource) [reply test/tribes.js:364]: yes that's part of the anti-gaslighting requirement: people should know that they got removed from the group, without wondering if they were in it in the first place

File: package.json [hunk 3]
```diff
@@ -18,7 +18,7 @@
     "pull-stream": "^3.6.14",
     "private-group-spec": "^2.1.1",
     "ssb-bfe": "^3.7.0",
-    "ssb-keyring": "^5.0.0",
+    "ssb-keyring": "https://gitlab.com/ahau/lib/ssb-keyring/-/archive/336a0e90f956e48fd5a44bff3da308664b1f3a65/ssb-keyring-336a0e90f956e48fd5a44bff3da308664b1f3a65.tar.gz",
```
[the remark this answers is not in the document]
Comment (Powersource) [reply package.json:21]: nope that doesn't work with pnpm + gitlab last i tried (or maybe i didn't try `gitlab:` but i tried like literally everything else)

**Pull request merged**